operator/src/main/java/org/keycloak/operator/controllers/KeycloakController.java

```diff
@@ -127,6 +127,7 @@ public UpdateControl<Keycloak> reconcile(Keycloak kc, Context context) {
         kcIngress.createOrUpdateReconciled();
 
         var status = statusBuilder.build();
+        status.setInstances(kcDeployment.getReadyReplicas());
```

Comment:
I'd like to keep the builder pattern here to make it consistent with how we set conditions, the status shouldn't be modified directly.

Also, I think we should set this inside `KeycloakDeployment.updateStatus`, not in controller, to make it consistent as well.

Reply:
I had started with the approach, but it seemed perhaps too involved, so I erred on the side of the narrowest change.

It's been updated now to create the sundrio builder and wrap that with an aggregator - over time the may be more to the status that will need to be constructed (the observedGeneration would be good to add as well) and it would be good to have the default building logic available.